### ISSUE TITLE
Documents only displaying "Generating PlantUML diagram" with release 1.5.1 update

### ISSUE THREAD

**Issue by josephpo** [joethei/obsidian-plantuml#22]
### What happened?

Post 1.5.1 upgrade all diagrams are not rendering only leaving a message, "Generating PlantUML diagram. The issue persists when creating a simple markdown with only the PlantUML tags and simple diagram.

### Obsidian version

v0.13.23

### Plugin version

1.5.1

### What distribution are you seeing the problem on?

Desktop

### Are you using a local `.jar` or a remote server

server

### Relevant log output

```shell
app.js:1 Obsidian Developer Console
main.ts:103 Dataview: Version 0.4.26 Loaded
main.ts:20 Loading natural language date parser plugin
main.ts:31 Loading Note Refactor plugin
main.ts:30 loading markdown-table-editor plugin
VM229:5309 loading plugin plantuml
index.ts:154 Dataview: Task & metadata parsing queued in 0.066s.
VM229:66 Uncaught (in promise) undefined
rejected @ VM229:66
Promise.then (async)
step @ VM229:69
eval @ VM229:70
__async @ VM229:54
DebouncedProcessors.png @ VM229:5211
e @ app.js:1
enter @ app.js:1
t.iterate @ app.js:1
update @ app.js:1
(anonymous) @ app.js:1
hr @ app.js:1
t @ app.js:1
t.applyTransaction @ app.js:1
get @ app.js:1
t.update @ app.js:1
_dispatch @ app.js:1
t.dispatch @ app.js:1
t.set @ app.js:1
e.setViewData @ app.js:1
e.setData @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
s @ app.js:1
Promise.then (async)
l @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.loadFileInternal @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.onLoadFile @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.onLoadFile @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.loadFile @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.setState @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.setState @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
s @ app.js:1
Promise.then (async)
l @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.setViewState @ app.js:1
eval @ VM191:74
o @ VM191:23
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.openFile @ app.js:1
e.onFileClick @ app.js:1
s @ app.js:1
Show 32 more frames
VM229:66 Uncaught (in promise) undefined
rejected @ VM229:66
Promise.then (async)
step @ VM229:69
eval @ VM229:70
__async @ VM229:54
DebouncedProcessors.png @ VM229:5211
e @ app.js:1
enter @ app.js:1
t.iterate @ app.js:1
update @ app.js:1
(anonymous) @ app.js:1
hr @ app.js:1
t @ app.js:1
t.applyTransaction @ app.js:1
get @ app.js:1
t.update @ app.js:1
_dispatch @ app.js:1
t.dispatch @ app.js:1
blur @ app.js:1
t.runCustomHandlers @ app.js:1
(anonymous) @ app.js:1
e.setEphemeralState @ app.js:1
e.setEphemeralState @ app.js:1
e.setEphemeralState @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
s @ app.js:1
Promise.then (async)
l @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.setViewState @ app.js:1
eval @ VM191:74
o @ VM191:23
checkCallback @ app.js:1
SR @ app.js:1
t.executeCommandById @ app.js:1
eval @ VM191:74
o @ VM191:23
t.onTrigger @ app.js:1
t.handleKey @ app.js:1
t.onKeyEvent @ app.js:1
index.html:1 Uncaught (in promise) undefined
Promise.then (async)
h @ app.js:1
t.onRender @ app.js:1
requestAnimationFrame (async)
Be @ app.js:1
t.queueRender @ app.js:1
t.set @ app.js:1
e.set @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
s @ app.js:1
Promise.then (async)
l @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.setMode @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.setState @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
D @ app.js:1
e.setViewState @ app.js:1
eval @ VM191:74
o @ VM191:23
checkCallback @ app.js:1
SR @ app.js:1
t.executeCommandById @ app.js:1
eval @ VM191:74
o @ VM191:23
t.onTrigger @ app.js:1
t.handleKey @ app.js:1
t.onKeyEvent @ app.js:1
```

**Comment by josephpo** [comment]
The same markdown file opened and parsed fine using VSC as a test.

**Comment by joethei** [comment]
I cannot reproduce this at all, can you give me any more information?

**Comment by xconnected** [comment]
Have the same - try planuml-svg for me this helped

**Comment by josephpo** [comment]
Let me know what you specifically want from my environment that will help you troubleshoot the issue e.g. logs, build, etc... I reverted to 1.4.0 and its working. Running macOS Big Sure version 11.6.4. Obsidian plugins installed are advanced tables, calendar, checklist, dataview, diagrams, kanban, natural language dates, note refractor, periodic notes, plantuml, quick switcher++,sliding panes, tag wrangler, templater, and workspaces plus.  Joe

**Comment by joethei** [comment]
The latest update (1.5.2) might have fixed this issue, maybe.

If it did not please let me know your debug information(Run the `Show debug info` command)

**Comment by ChienChunCHENG** [comment]
Thanks for the update (1.5.2). It has fixed another issue for Pandoc. ;)

**Comment by josephpo** [comment]
Thank you for the update, it has resolved my issue in 1.5.2 so can be closed.
